<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOSpreadsheetWin</class>
 <widget class="QDialog" name="IOSpreadsheetWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spreadsheet 0</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Show0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Show1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Show2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Show3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="SheetClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SheetCopy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SheetPaste">
       <property name="text">
        <string>Paste</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RowAdd">
       <property name="text">
        <string>Row add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RowRem">
       <property name="text">
        <string>Row rem</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ColAdd">
       <property name="text">
        <string>Col add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ColRem">
       <property name="text">
        <string>Col rem</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="SpreadsheetScreen">
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
